web/src/pages/kiosk/[action_id].tsx

```diff
@@ -61,9 +69,20 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
   if (data?.action[0].action === "") {
     return { props: { error_code: "no_sign_in" } };
   }
+  const { app_metadata, verified_app_metadata, ...app_data } =
+    data.action[0].app;
+  const app_metadata_item = verified_app_metadata?.[0] ?? app_metadata?.[0];
```

Comment:
not fully correct, we should not pass image URLs that don't come from a verified app